resteasy-feature-pack/galleon-preview-feature-pack/pom.xml

```diff
@@ -96,6 +96,12 @@
                                     </includes>
                                     <filtering>true</filtering>
                                 </resource>
+                                <resource>
+                                    <directory>${common.resources.directory}</directory>
+                                    <excludes>
+                                        <include>configs/**/model.xml</include>
```

Comment:
Should it be <exclude> ?

Reply:
No, it should be `includes`. I just copied a code snippet and didn't change it all :)